bisontry-messenger/values.yaml

```diff
@@ -0,0 +1,12 @@
+global:
+  name: bisontry-messenger
+  namespace: orakl
+
+image:
+  repository: asia-southeast1-docker.pkg.dev/bisontry/bisontry/bisontry-messenger
```

Comment:
Is the bisontry/bisontry correct?

Reply:
Yes! first `bisontry` is GCP project name and second one is registry name. 😄